src/app/containers/Include/index.test.jsx

```diff
@@ -157,4 +158,36 @@ describe('IncludeContainer', () => {
     expect(mockIdt2Amp).not.toHaveBeenCalled();
     expect(loggerMock.info).not.toHaveBeenCalled();
   });
+
+  it('should render a VJ include on an Amp page with toggles enabled', () => {
+    const vjProps = {
+      ampSrc: 'https://news.files.bbci.co.uk/vj.amp',
+      ampImage:
+        'https://news.files.bbci.co.uk/vj/?amp-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-img.png',
```

Comment:
Is this a real `ampImage` value, it looks a bit strange? Please correct me if I'm wrong!
```suggestion
        'https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_ws_640_7ania.png',
```